src/api/EatWell.Web/Controllers/ProductController.cs

```diff
@@ -12,38 +12,38 @@ public class ProductController : ControllerBase
     {
         private readonly IProductService _productService;
 
-        [HttpPost]
-        public IActionResult CreateProduct(ProductModel product)
-        {
-            _productService.CreateProduct(product);
-            return Ok();
-        }
-
         public ProductController(IProductService productService)
         {
             _productService = productService;
         }
 
-        [HttpDelete("{id:int}")]
-        public IActionResult DeleteProduct(int id)
-        {
-            _productService.DeleteProduct(id);
-
-            return Ok();
-        }
-
         [HttpGet]
         public IActionResult GetProducts()
         {
             var products = _productService.GetProducts();
             return Ok(products);
         }
 
+        [HttpPost]
+        public IActionResult CreateProduct(ProductModel product)
+        {
+            _productService.CreateProduct(product);
+            return Ok();
+        }
+
         [HttpPut]
```

Comment:
We need to know which product to update - so we should get the id in url.
I.e : [HttpPut("{idPost:int}")]